docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,686 @@
+      plan: "developer.x1"
+      billing_cloud_account_id: "123456789"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      kafka_req_resp
+----
+. In the `name` field, enter a name for the Kafka instance.
+. In the `billing_cloud_account_id`, enter the billing cloud account ID.
+. In the `openshift_offline_token` field, enter your OpenShift offline token.
++
+All other information for the instance is provided by the Kafka APIs.
++
+When you run the `create_kafka_` module as part of the playbook at the end of this procedure, Ansible saves the output of that command in a variable in the `register` field. In the preceding example, Ansible saves the created Kafka instance as `kafka_req_resp`.
+
+. To connect your applications or services to a Kafka instance in {product-kafka}, you must first create a service account with credentials. To add a module that creates a service account, copy the `create_service_account` module shown in the following example and paste it into the `rhoas_kafka.yml` file.
++
+.Example `create_service_account` module
+[source,yaml]
+----
+- name: Create Service Account
+    create_service_account:
+      name: "service-account-name"
+      description: "This is a description of the service account"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      srvce_acc_resp_obj
+----
+. Enter values for the `name`, `short description`, and `openshift_offline_token` fields.
++
+When you run the `create_service_account` module as part of the playbook at the end of this procedure, Ansible populates the generated service account credentials in the `client_id` and `client_secret` fields in the terminal after it creates the service account.
+. To create Access Control List (ACL) permissions for the service account and bind that ACL to the Kafka instance, copy the `create_kafka_acl_binding` module shown in the following example and paste it in your `rhoas_kafka.yml` file.
++
+.Example `create_kafka_acl_binding` module
+[source,yaml]
+----
+- name: Create kafka ACL Service Binding
+    rhoas.rhoas.create_kafka_acl_binding:
+      kafka_id: "{{ kafka_req_resp.kafka_id }}"
+      # To hard code the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+      principal: " {{ srvce_acc_resp_obj['client_id'] }}"
+      # To hard code the principal_id, uncomment and use the following line:
+      # principal: "PRINCIPAL_ID"
+      resource_name: "topic-name"
+      resource_type: "Topic"
+      pattern_type: "PREFIXED"
+      operation_type: "all"
+      permission_type: "allow"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register: kafka_acl_resp
+
+----
+. You can allow Ansible to get the Kafka ID from the `kafka_req_resp` variable or enter it in the `kafka_id` field.
+. In the `openshift_offline_token` field, enter your OpenShift offline token.
++
+The following list is a description of each field that must have a value in an ACL binding module.
```

Comment:
I'm not sure this list logically follows as a result of the preceding step. It might be better as its own step. It seems that a user might want to change some, but not all, of these values? Therefore, you could leave the wording a little open, like the following?

"Consider whether you need to  specify your own value for any of the fields in the following list. These fields must all have values in an ACL binding module."